src/Driver/Postgres/Schema/PostgresColumn.php

```diff
@@ -42,80 +43,167 @@ class PostgresColumn extends AbstractColumn
         'attributes',
     ];
 
+    private const INTERVAL_TYPES = [
+        'YEAR',
+        'MONTH',
+        'DAY',
+        'HOUR',
+        'MINUTE',
+        'SECOND',
+        'YEAR TO MONTH',
+        'DAY TO HOUR',
+        'DAY TO MINUTE',
+        'DAY TO SECOND',
+        'HOUR TO MINUTE',
+        'HOUR TO SECOND',
+        'MINUTE TO SECOND',
+    ];
+
+    private const INTERVALS_WITH_ALLOWED_PRECISION = [
+        'SECOND',
+        'DAY TO SECOND',
+        'HOUR TO SECOND',
+        'MINUTE TO SECOND',
+    ];
+
+    protected array $aliases = [
+        'int'            => 'integer',
+        'smallint'       => 'smallInteger',
+        'bigint'         => 'bigInteger',
+        'incremental'    => 'primary',
+        'bigIncremental' => 'bigPrimary',
+        'bool'           => 'boolean',
+        'blob'           => 'binary',
+        'bitVarying'     => 'bit varying',
```

Comment:
Need to add all magic types as `@method` annotations for the class.

Reply:
Added